README.md

```diff
@@ -1200,6 +1200,12 @@ Language: English
 Notable Channels: `#general`, `#show`, `#help`, `#career`, `#data-science` `#resources`, `#architecture`, `#cloud`, `#databases`, `#etl-workflow` \
 Language: English
 
+<img align="left" height="94px" width="94px" alt="Server Icon" src="images/server_icons/eddiejaoudecommunity.webp" />
+
+[__Eddie Jaoude Community__](https://discord.com/invite/jZQs6Wu) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.webp">](http://eddiejaoudecommunity.github.io) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.webp">](http://github.com/EddieJaoudeCommunity) \
```

Comment:
Hey @eddiejaoude thanks for opening the PR. Based on your community description, since you're the owner, your community is very much eligible to be assigned as [`official`](https://github.com/mhxion/awesome-discord-communities/blob/main/badges.md#official-identification-badge). Simply adding this after the name `[<img height="16px" width="16px" alt="Official Badge" src="images/badges/official.webp">](badges.md#official-identification-badge)` will show the official badge.

Everything else looks okay 👍

Reply:
Will do 👍